quantum/split_common/transport.c

```diff
@@ -16,6 +16,7 @@
 
 #include <string.h>
 #include <debug.h>
+#include <timer.h>
```

Comment:
Cant see any `timer_` call sites in thif file?
```suggestion
```

Reply:
Oops, that's a remnant from moving things around...